client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -208,7 +209,7 @@ class Settings extends React.Component {
 		return (
 			<div className="email-settings__container">
 				<ExtendedHeader label={ translate( 'Origin' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **74** times:
`translate( 'Original' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/store-email-settings-origin-ui-adjustment)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).